### ISSUE TITLE
ServiceNotFoundException

### ISSUE THREAD

**Issue by quentin-st** [irozgar/gulp-rev-versions-bundle#9]
Hi,

I'm running Symfony 3.1. I followed the instructions in the README, but stumble upon the following exception when running `server:run`:

```
[Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException]                                                             
The service "assets.packages" has a dependency on a non-existent service "irozgar_gulp_rev_versions.asset.gulp_rev_version_strategy". 
```

I tried registering this service in my own `services.yml`: `server:run` succeeds, but then I get a `An exception has been thrown during the rendering of a template ("The file "assets/css/lib.css" does not exist and there is no version file for it") in AppBundle::base.html.twig at line 45.` exception.

```
services:
    irozgar_gulp_rev_versions.asset.gulp_rev_version_strategy:
        class: Irozgar\GulpRevVersionsBundle\Asset\GulpRevVersionStrategy
        arguments:
            - "%kernel.root_dir%"
            - "%gulp_rev_manifest_path%"
```

> Note: after a bit of debugging, I found that your bundle handles framework bundle's assets, but not mine

**Comment by irozgar** [comment]
Can you paste the contents of your manifest file?

**Comment by quentin-st** [comment]
Sure I can! My rev-manifest.json is located in the default location (app/Resources/assets/)

```
{
  "s.js": "s-9de8ec1ad8.js"
}
```

I added some var_dumps in `GulpRevVersionStrategy` (especially in `getVersion` and `applyVersion`): it breaks assets URLs in Symfony's exceptions pages (looks like ̀string[30]...`), but not on my own pages. It seems like your bundle is only active in framework's one, but not mine. Have I missed anything?

**Comment by irozgar** [comment]
The file that is not found is not in the manifest, so I that file doesn't have a version file. Is this right?

Your assets should be in the web directory (you can have them in your bundle public directory, but you have to copy or symlink them in the web directory with the command _assets:install_. More info about the directory for public files in bundles [here](https://symfony.com/doc/2.8/bundles.html#bundle-directory-structure)).

If you have already done this tell me where is located the file _assets/css/lib.css_ and this afternoon I'll try to figure why it's not working.

**Comment by quentin-st** [comment]
Sorry, I made a lot of tests so the manifest I posted isn't the one I made tests with yesterday.
I cut done my assets list to a single one to make it easier to debug.

My gulped assets are located in `web/assets/*`. In my latest test, I have a single file located in `web/assets/js/s.js`. I reference it in the Twig template using `{{ asset('assets/js/s.js') }}`.

**Comment by irozgar** [comment]
And whats the exception message for this latest test? Is it _The file "assets/js/s.js" does not exist and there is no version file for it_?

If the message is this one can you check the value of [the variable $fileExists in this line](https://github.com/irozgar/gulp-rev-versions-bundle/blob/master/Asset/GulpRevVersionStrategy.php#L68). And if it's false could you put `var_dump($this->kernelRootDir.'/../web/'.$path)` one line above or under that line? This should print the path of the file relative to your app folder. Is this path correct?

Reading your comment I think everything be working.

**Comment by quentin-st** [comment]
About the exception, that was actually a problem in my manifest, my bad... (I'm new to this whole gulp thing)

I still have to manually define the `irozgar_gulp_rev_versions.asset.gulp_rev_version_strategy` service in my own `services.yml` file or else the exception mentioned in the first message is triggered:

```
The service "assets.packages" has a dependency on a non-existent service "irozgar_gulp_rev_versions.asset.gulp_rev_version_strategy".
```

**Comment by irozgar** [comment]
I think I've found the problem.
There's an error in the readme with the name of the package. The correct composer require is `composer require irozgar/gulp-rev-versions-bundle`.

So the steps to fix this problem are:
1. Remove irozgar/gulp-dev-versions entry in the require section of the composer.json file.
2. Remove vendor/irozgar/gulp-dev-versions-bundle directory and its content. 
3. Run `composer require irozgar/gulp-rev-versions-bundle`.

I made a typo writing gulp-dev-versions-bundle instead of gulp-rev-versions-bundle in some places (autocompletion and copy/paste made me notice the typo after the package was in packagist) and despite I deleted the package with the wrong name in packagist, it seems it can still be installed.

I'll update the README to fix this.
